x-pack/plugins/osquery/cypress/e2e/all/live_query_run.cy.ts

```diff
@@ -80,37 +76,34 @@ describe('ALL - Live Query run custom and saved', { tags: ['@ess', '@serverless'
     cy.getBySel(RESULTS_TABLE).within(() => {
       cy.getBySel(RESULTS_TABLE_BUTTON).should('exist');
     });
-    cy.react(RESULTS_TABLE_CELL_WRRAPER, {
-      props: { id: 'message', index: 1 },
-    }).should('exist');
-    cy.react(RESULTS_TABLE_CELL_WRRAPER, {
-      props: { id: 'osquery.days.number', index: 2 },
-    })
-      .react('EuiIconTip', { props: { type: 'indexMapping' } })
-      .should('exist');
+    cy.get('[data-gridcell-column-index="1"][data-test-subj="dataGridHeaderCell-message"]').should(
+      'exist'
+    );
+    cy.get(
+      '[data-gridcell-column-index="2"][data-test-subj="dataGridHeaderCell-osquery.days.number"]'
+    )
+      .should('exist')
+      .within(() => {
+        cy.get(`.euiToolTipAnchor`);
+      });
   });
 
   it('should run customized saved query', () => {
     cy.contains('New live query').click();
     selectAllAgents();
-    cy.react('SavedQueriesDropdown').type(`${savedQueryName}{downArrow}{enter}`);
+    cy.getBySel(SAVED_QUERY_DROPDOWN_SELECT).type(`${savedQueryName}{downArrow}{enter}`);
     inputQuery('{selectall}{backspace}select * from users;');
     cy.wait(1000);
     submitQuery();
     checkResults();
     navigateTo('/app/osquery');
-    cy.react('EuiButtonIcon', { props: { iconType: 'play' } })
-      .eq(0)
-      .should('be.visible')
-      .click();
+    cy.get('[aria-label="Run query"]').eq(0).should('be.visible').click();
 
     cy.get(LIVE_QUERY_EDITOR).contains('select * from users;');
   });
 
   it('should open query details by clicking the details icon', () => {
-    cy.react('EuiButtonIcon', { props: { iconType: 'visTable' } })
-      .first()
-      .click();
+    cy.get('[aria-label="Details"]').eq(0).should('be.visible').click();
```

Comment:
nit: why do you change `.first()` to `.eq(0)`? I'd say the first (which is the `.first()` 😈) is easier on the eyes

Reply:
Good catch, I think I have been more used to use eq recently, and it came automatically. I can adjust to .first() if you find it more readable :)